scripts/sct_run_batch.py

```diff
@@ -0,0 +1,159 @@
+                'The processing script (task) should accept a subject directory as its only argument. '
+                'Additional information is passed via environment variables and the arguments '
+                'passed via `--task-args`',
+  formatter_class = SmartFormatter,
+  prog=os.path.basename(__file__).strip('.py'))
+
+parser.add_argument("--jobs", type = int, default = 1,
+                    help = 'The number of jobs to run in parallel. '
+                           'either an integer greater than or equal to one '
+                           'specifying the number of cores, 0 or a negative integer '
+                           'specifying number of cores minus that number. For example '
+                           '--jobs -1 indicates run ncores - 1 jobs in parallel.')
+parser.add_argument("--path-data", help = 'Setting for environment variable: PATH_DATA\n'
+                                          'Path containing subject directories in a consistent format')
+parser.add_argument("--subject-prefix", default = "sub-",
+                    help = 'Subject prefix, defaults to "sub-"')
+parser.add_argument("--path-output", default = "./",
+                    help = 'Base directory for environment variables:\n'
+                           'PATH_RESULTS=' + os.path.join('<path-output>', 'results') + '\n'
+                           'PATH_QC=' + os.path.join('<path-output>', 'QC') + '\n'
+                           'PATH_LOG=' + os.path.join('<path-output>', 'results') + '\n'
+                           'Which are output paths for results, logs, and QC')
+parser.add_argument("--include",
+                    help = 'Optional regex used to filter the list of subject directories. Only process '
+                           'a subject if they match the regex. Inclusions are processed before exclusions')
+parser.add_argument("--exclude",
+                    help = 'Optional regex used to filter the list of subject directories. Only process '
+                           'a subject if they do not match the regex. Exclusions are processed '
+                           'after inclusions')
+parser.add_argument("--path-segmanual", default = ".",
+                    help = 'Setting for environment variable: PATH_SEGMANUAL\n'
+                           'A path containing manual segmentations to be used by the task program.')                
+parser.add_argument("--itk-threads", default = 1,
+                    help = 'Setting for environment variable: ITK_GLOBAL_DEFAULT_NUMBER_OF_THREADS\n'
+                           'Number of threads to use for ITK based programs including ANTs. Set to a low '
+                           'number to avoid a large increase in memory. Defaults to 1')
+parser.add_argument("--task-args", default = "",
+                    help = 'A quoted string with extra flags and arguments to pass to the task script. '
+                           'The constructed call will look like <task> <subject> <task-args>')
```

Comment:
it is not clear what "subject" means here. Also, it would be nice to be explicit and exhaustive about what are the possible values for task-args

Reply:
`task-args` is in response to your suggestion above to allow the user to pass extra flags, I suspect this will generally be unused, but maybe we could clarify with a concrete example? An exhaustive list is impossible, given that we don't know in advance what script is being run and what arguments it might require.

Reply:
ah! ok i didn't understand at first what this flag was for. I've suggested a rewording.